sdk/identity/ci.yml

```diff
@@ -73,12 +73,14 @@ extends:
     LiveTestStages:
       - template: /eng/pipelines/templates/stages/archetype-sdk-tests-isolated.yml
         parameters:
+          PersistOidcToken: true
+          Location: 'westus2'
           CalledFromClient: true
           ServiceDirectory: identity
           SupportedClouds: 'Public,UsGov,China'
-          UseFederatedAuth: false
-          EnvVars:
-            SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+#          EnvVars:
+#            AZ_TENANT_ID: $(IDENTITY_TENANT_ID)
```

Comment:
You will need to map `ARM_OIDC_TOKEN: $(ARM_OIDC_TOKEN)` into the env to use it at test runtime.